docs/development.rst

```diff
@@ -29,13 +29,10 @@ For more options for building the web client, run: ::
 Vagrant
 ^^^^^^^
 
-A shortcut to going through the installation steps for development is to use
-`Vagrant <https://www.vagrantup.com>`_ to setup the environment on a
-`VirtualBox <https://www.virtualbox.org>`_ virtual machine. To setup this
-environment run ``vagrant up`` in the root of the repository. This will spin up
-and provision a virtual machine, provided you have Vagrant and VirtualBox
-installed. Vagrant uses `Ansible <https://ansible.com>`_ for provisioning Girder and its various
-dependencies.
+A shortcut to going through the development environment configuration steps is
```

Comment:
In my opinion, we should push new developers towards using Vagrant as the preferred development environment. That would necessitate that we move this section up within this page, and reword some of the `Configuring Your Development Environment` section too.

@sutartmelson If you'd like to give that a shot, feel free. Otherwise, let me know, and I'll be happy to tackle this bit myself.

Reply:
I think changing the existing documentation any more than is already happening collides too much with #2257. Further changes should probably be added there. Maybe @zachmullen has an opinion about it?

Reply:
Agreed. Let's leave it alone.